tests/unittests/tests-gnrc_mac_internal/Makefile.include

```diff
@@ -0,0 +1,2 @@
+USEMODULE += gnrc_priority_pktqueue
+USEMODULE += gnrc_mac
```

Comment:
I think the tests would benefit if you set the CFLAGS according to the size defines you use in these tests.